src/suresh/StringsIdenticalwithoutArray.java

```diff
@@ -0,0 +1,24 @@
+package suresh;
+
+public class StringsIdenticalwithoutArray {
+
+		void identicalStringFinder() {
+			/*String s1 = "technoCredits";
+			String s2 = "teChnoCreditS";*/
+			
+			String s1 = "pune";
+			String s2 = "pune";
+			if(s1==s2) {
```

Comment:
make parametrized method
you need to use for loop and compare single char with another string char